admin/includes/languages/english/lang.plugin_manager.php

```diff
@@ -39,7 +40,7 @@
     'TEXT_CONFIRM_ENABLE' => 'Are you sure you want to enable this plugin?',
     'TEXT_INFO_UPGRADE' => 'Please select the version you want to upgrade to.',
     'TEXT_INFO_UPGRADE_CONFIRM' => 'Upgrade version %s',
-    'TEXT_INFO_UPGRADE_WARNING' => 'Warning: blah blah',
+    'TEXT_INFO_UPGRADE_WARNING' => '',
```

Comment:
Perhaps in the future, plugins could inject their own warnings?  Could be a powerful feature.